### ISSUE TITLE
build and release a version supporting .net standard

### ISSUE THREAD

**Issue by digitalpowers** [mattwar/iqtoolkit#19]
Any chance you will release this built for .net standard?

**Comment by mattwar** [comment]
Fixed by merging in standard branch.  The core parts are now in .net standard.  Some specific providers continue to be .Net Framework due to associate libraries. Not yet published.

**Comment by digitalpowers** [comment]
@mattwar were you planning to release a version to nuget that supported .net standard? I understand some specific parts wouldnt be included because they are framework specific, but it would help to have the library be nugettable, :) also thanks for the tool and the series, it was fascinating and fun.

**Comment by mattwar** [comment]
Yes, I'm planning to eventually push out a version 2.0 that is .net standard to nuget.
